content/wasm-languages/javascript.md

```diff
@@ -26,6 +25,80 @@ There are three popular ways of building JavaScript into WebAssembly.
 
 Recently, Suborbital has introduced a version of [Javy](https://github.com/suborbital/javy) that supports some of their extensions.
 
+An experimental SDK for building Spin apps using JavaScript (and TypeScript) was [introduced in a Fermyon blog article](https://www.fermyon.com/blog/spin-js-sdk) back in December 2022. This [Spin SDK for Javascript](https://github.com/fermyon/spin-js-sdk) borrows heavily from Javy, using the same approach of providing a CLI utility to convert a JS file into a Wasm file.
```

Comment:
Maybe out of scope for this update but we should mention `jco`

Reply:
Added a jco sentence. Yes, we cal collaborate on that much more in future iterations of this page. Super interesting.